assembly/utils.ts

```diff
@@ -249,3 +249,11 @@ export function atou128(str: string, radix: i32 = 0): u128 {
 
   return isNeg ? -result : result;
 }
+
+export function LOAD<T>(arr: ArrayBuffer, offset: usize = 0): T {
```

Comment:
It seems LOAD and STORE polyfills don't need anymore

Reply:
What's the preferred way to use it now?

Reply:
just `load` and `store`. This PR already has replacments